yarn.lock

```diff
@@ -21309,6 +21404,16 @@ react-native@0.64.0:
   languageName: node
   linkType: hard
 
+typescript@>3:
```

Comment:
should probably dedupe this one to the 4.1.5 we're rocking here to avoid weird clashes? (TS should probably be a peer dep or something)

Reply:
This is intentionally super loose on our side, with the goal being to not include typescript twice for folks but I guess that's just unreasonable in the node ecosystem